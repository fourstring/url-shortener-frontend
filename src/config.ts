import {globalE2EMockClient} from './mocks/e2eClient'

let config: Record<string, any> = {};
if (process.env["CI"]) {
  config = {
    baseURL: 'https://api.fourstring.dev', // The base url of the backend project
    globalE2EMock: true,
    globalE2EMockClient: globalE2EMockClient,
    jwtMonitorRate: 60000, //ms
    jwtRefreshThreshold: 120, //s
    shortenLinkBaseURL: 'https://api.fourstring.dev/s'
  }
} else {
  config = {
    baseURL: 'https://api.fourstring.dev', // The base url of the backend project
    globalE2EMock: true,
    globalE2EMockClient: globalE2EMockClient,
    jwtMonitorRate: 60000, //ms
    jwtRefreshThreshold: 120, //s
    shortenLinkBaseURL: 'https://api.fourstring.dev/s'
  }
}
export default config;
