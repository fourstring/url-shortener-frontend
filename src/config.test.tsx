import config from './config';

/*
* 检查 config 参数
* 为 config 添加快照，测试 config 状态不变
* @author wfn
*/
test('test config', () => {
  expect(config).toMatchSnapshot();
});