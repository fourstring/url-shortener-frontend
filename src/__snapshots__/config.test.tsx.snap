// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test config 1`] = `
Object {
  "baseURL": "http://localhost:8080",
}
`;
